ServerCore/Pages/Teams/Delete.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    public class DeleteModel : EventSpecificPageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
+
+        public DeleteModel(ServerCore.Models.PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Team Team { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int? id)
+        {
+            if (id == null)
```

Comment:
if you make id non-nullable, will the NotFound get handled automatically by the path not matching?

Reply:
Yes - the CRUD wizard adds this, and we can clean it up.